content/user/advanced-modules/KnowledgeBase.adoc

```diff
@@ -0,0 +1,74 @@
+---
+Title: Knowledge Base
+Weight: 205
+---
+
+:experimental:
```

Comment:
We usually write this line like this:

`:experimental:   ////this is here to allow btn:[]syntax used below`

I would prefer to keep that comment in, that people are reminded that they don't need that line if they don't use `btn`